src/vif.jl

```diff
@@ -0,0 +1,119 @@
+# putting this behind a function barrier means that the model matrix
+# can potentially be freed immediately if it's large and constructed on the fly
+function _find_intercept(model::RegressionModel)
+    modelmat = modelmatrix(model)
+    # XXX collect is necessary for Julia 1.6
+    # but it's :just: an array of references to views, so shouldn't be too
+    # expensive
+    return findfirst(Base.Fix1(all, isone), collect(eachcol(modelmat)))
+end
+
+_find_intercept(form::FormulaTerm) = _find_intercept(form.rhs)
+# we need these in case the RHS is a single term
+_find_intercept(::AbstractTerm) = nothing
+_find_intercept(::InterceptTerm{true}) = 1
+_find_intercept(m::MatrixTerm) = _find_intercept(m.terms)
+function _find_intercept(t::TupleTerm) 
+    return findfirst(Base.Fix2(isa, InterceptTerm{true}), t)
+end
+
+# borrowed from Effects.jl
+function get_matrix_term(x)
+    x = collect_matrix_terms(x)
+    x = x isa MatrixTerm ? x : first(x)
+    x isa MatrixTerm || throw(ArgumentError("couldn't extract matrix term from $x"))
+    if first(x.terms) isa MatrixTerm
+        x = only(x.terms)
+    end
+    return x
+end
+
+"""$(@doc StatsAPI.vif)"""
```

Comment:
If we repeat the docstring it will be shown twice, right?

Reply:
This is the only way I found to get it to show up in the manual here.

Reply:
[Apparently](https://stackoverflow.com/questions/70137119/how-to-include-the-docstring-for-a-function-from-another-package-in-my-julia-doc) you can pass `modules=[StatsModels, StatsAPI]` to `makedocs`. This will probably have the side effect of printing warnings for StatsAPI docstrings that are not in the manual though, but that's less of an issue IMO.

Reply:
oh very nice, done